common-app-lib/src/cmr/common_app/services/search.clj

```diff
@@ -58,20 +55,20 @@
 (defn validate-query
   "Validates a query model. Throws an exception to return to user with errors.
   Returns the query model if validation is successful so it can be chained with other calls."
-  [context query]
+  [_context query]
   (if-let [errors (seq (qv/validate query))]
     (errors/throw-service-errors :bad-request errors)
     query))
 
 (defmulti search-results->response
   "Converts query search results into a string response."
-  (fn [context query results]
+  (fn [_context query _results]
```

Comment:
I have been converting the defmulti/defmethod functions to a function with a case statement when possible (it is almost always possible since most of the time all of the defmethod functions are in the same namespace), but I will let you decide if you want to do that.